packages/components/vite.config.ts

```diff
@@ -1,10 +1,11 @@
 /* (c) Copyright Frontify Ltd., all rights reserved. */
 
 import react from '@vitejs/plugin-react';
-import { defineConfig } from 'vitest/config';
+import tsConfigPaths from 'vite-tsconfig-paths';
+import { type UserConfig, defineConfig } from 'vitest/config';
 
 export default defineConfig({
-    plugins: [react()],
+    plugins: [react(), tsConfigPaths()] as UserConfig['plugins'],
```

Comment:
```suggestion
    plugins: [react(), tsConfigPaths()],
```

Reply:
all plugin cause this

 Type 'PluginOption[]' is not assignable to type 'PluginOption

Reply:
why do you dislike the array type?

Reply:
@SamuelAlev the issues persists for me